Formula/emqx.rb

```diff
@@ -38,13 +38,15 @@ class Emqx < Formula
   def install
     ENV["PKG_VSN"] = version.to_s
     touch(".prepare")
-    system "make", "emqx-rel"
-    prefix.install Dir["_build/emqx/rel/emqx/*"]
-    rm %W[
-      #{bin}/emqx.cmd
-      #{bin}/emqx_ctl.cmd
-      #{bin}/no_dot_erlang.boot
```

Comment:
maybe re-add this line?

Reply:
`bin/emqx.cmd` and `bin/emqx_ctl.cmd` are deleted in the block below where I unpack `tar.gz`, and I explained the issue with `no_dot_erlang.boot` in another comment.